### ISSUE TITLE
Delete watched playlist videos

### ISSUE THREAD

**Issue by cameronetchart** [Hebo/chompy#16]
### **tl;dr -- option to auto remove videos that were downloaded via PLAYLIST_SYNC but then removed from that playlist?**

One of the ways I use chompy is to set my YouTube watch later playlist to auto download, then I watch the video, then remove it from youtube's watch later. But then, all the videos remain in my chompy browse the directory in VLC or a file browser and directly watch the mp4's that were downloaded, allowing me to directly watch the videos without having to always go through the youtube app and Shortcuts. Then I later remove the video from my watch later playlist on youtube. But then the video stays in my chompy directory forever, unless I use the max size limit option.

I saw that chompy saves downloaded playlist files to a ".ytdl-archive.txt" file but does not save to this file if the user activates chompy via iOS shortcuts. 

The logic would be, at time of refreshing the PLAYLIST_SYNC playlist:
**IF (video is in .ytdl-archive.txt) AND (video is not in PLAYLIST_SYNC playlist) THEN (delete local file).** 

Thanks!

**Comment by Hebo** [comment]
Sorry for the slow reply, my github notifications are misconfigured.

Right now, Chompy is a fairly thin layer over youtube-dl, which has a lot of advantages for simplicity, and so I want to leverage the wide range of functionality in youtube-dl as much as possible rather than building heavy logic directly into Chompy.  

The `ytdl-archive.txt` is generated by [youtube-dl](https://github.com/ytdl-org/youtube-dl/blob/master/README.md#how-do-i-download-only-new-videos-from-a-playlist), but there doesn't seem to be an easy way to have youtube-dl handle deletions, from my quick search.

For my use case, I don't worry about deleting videos after watching them, instead allowing them to be automatically deleted via the MAX_SIZE setting. Can you give me more info on why deleting via MAX_SIZE doesn't work for you?

There still might be some good ways to approach deleting watched videos, but I want to better understand where the current solution falls short.
